Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/openshift/origin-release:golang-1.15 AS builder
+FROM registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux AS builder
```

Comment:
i would not change this since  it will avoid community  user to build the image. If we have a quota issue on docker hub, we may consider using golang image in grc.io